### PR TITLE
feat: SFX deploy

### PR DESCRIPTION
**Pull request by jrittenh**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
```
Comment (ialford) [inline .github/workflows/ci.yml:8]:
```suggestion
    branches-ignore: [ main ]
```

I recommend running on push to each branch, and then again on PR, but not on pushes to `main`. As it is, you won't know that a branch is failing tests until you do the PR, which is potentially wasting time for the committer and reviewer.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```
Comment (ialford) [inline .github/workflows/ci.yml:15]:
```suggestion
    runs-on: ubuntu-20.04
```

NIT: prefer a hardcoded version

File: README.md [hunk 3]
```diff
@@ -1,14 +1,3 @@
...
-
-## Useful commands
-
- * `npm run build`   compile typescript to js
- * `npm run watch`   watch for changes and compile
- * `npm run test`    perform the jest unit tests
- * `cdk deploy`      deploy this stack to your default AWS account/region
- * `cdk diff`        compare deployed stack with current state
- * `cdk synth`       emits the synthesized CloudFormation template
+Infrastructure code to deploy SFX code to hosted Ex Libris instance
```
Comment (ialford) [inline README.md:3]: Include an example of how to stand up the stack, including any context overrides that you may do when standing up in test vs production.

File: src/sfx-deploy-pipeline-stack.ts [hunk 4]
```diff
@@ -205,30 +221,18 @@ export class SfxDeployPipelineStack extends Stack {
...
-          value: ``,
-          type: codebuild.BuildEnvironmentVariableType.PLAINTEXT,
-        },
-      }
     })
 
     const slackApprovalAction = new ManualApprovalAction({
       actionName: 'ManualApproval',
-      additionalInformation: `A new version of https://github.com/ndlib/sfx has been deployed to https://>SFX_ADDRESS_HERE< and is awaiting your approval. If you approve these changes, they will be deployed to https://>SFX_ADDRESS_HERE<.`,
+      additionalInformation: `A new version of https://github.com/ndlib/sfx has been deployed to https://findtext.library.nd.edu/ndu_test and is awaiting your approval. If you approve these changes, they will be deployed to https://findtext.library.nd.edu/ndu_local.`,
```
Comment (ialford) [inline src/sfx-deploy-pipeline-stack.ts:235]: You may want to parameterize the URLs, particularly if you're constructing them somewhere else from the parameter store values - see [DEC Blueprints](https://github.com/ndlib/dec-blueprints/blob/main/src/buzz/buzz-pipeline.ts#L229) as an example

**Review by ialford — CHANGES_REQUESTED** [review]

**Review by jgondron — APPROVED** [review]

**Review by ialford — APPROVED** [review]

**Pull request merged**